ai_diffusion/workflow.py

```diff
@@ -179,6 +179,20 @@ def _sampler_params(
     return params
 
 
+def _apply_strength(strength: float, params: dict, fixed_steps: bool = False) -> dict[str, Any]:
```

Comment:
Hm, I'm not happy with this dict as in-out-parameter. Maybe keep the function, but return `tuple[int, int]`
```
(strength, steps, fixed_steps) -> (steps, start_at_step)
```
Then call it directly in `_sampler_params`. Maybe my suggestion with 2 functions was confusing, I didn't actually mean to separate these at the call site, just thought the step computation made a nice logical unit.

Reply:
I'll have pass the `strength` and `fixed_steps` parameters to `_sampler_params` if that's alright.

Reply:
Done.